tornettools/generate_tgen.py

```diff
@@ -10,22 +12,33 @@
 from tornettools.generate_defaults import *
 from tornettools.util import load_json_data
 
+from Crypto.PublicKey import RSA
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
+from cryptography.hazmat.backends import default_backend
+
 def generate_tgen_config(args, tgen_clients, tgen_servers):
     # make sure the config directory exists
     abs_conf_path = "{}/{}".format(args.prefix, CONFIG_DIRPATH)
     if not os.path.exists(abs_conf_path):
         os.makedirs(abs_conf_path)
 
-    server_peers = ["{}:{}".format(server['name'], TGEN_SERVER_PORT) for server in tgen_servers]
+    if args.hidden:
+        server_peers = ["{}:{}".format(server['hs_onion_url'], TGEN_SERVER_PORT) for server in tgen_servers]
```

Comment:
I propose to change this to `TGEN_HIDDENSERVICE_PORT`.

Reply:
Done.